src/plot_recipes.jl

```diff
@@ -153,15 +153,15 @@ julia> plot([B1, B2], 1e-4);
 
 ```
 """
-@recipe function plot_lazyset(arr::Vector{<:LazySet}, ε::Float64;
+@recipe function plot_lazyset(arr::Vector{S}, ε::Float64;
```

Comment:
while you are at this, `arr` is an ugly name, should we call it `array`, or `Xk`?

Reply:
done